docs/devices-api/more-info/cordova.mdx

```diff
@@ -24,7 +24,7 @@ pod install
 
 ## Usage
 
-To use our SDK go to your index.html add:
+To use our SDK go to your index.html and add:
```

Comment:
nit: wrap index.html around backticks to render as code, and comma after "To use our SDK"